awscli/examples/cognito-idp/tag-resource.rst

```diff
@@ -0,0 +1,11 @@
+**To tag a user pool**
+
+The following ``tag-resource`` example applies ``administrator`` and ``department`` tags to the requested user pool. ::
+
+    aws cognito-idp tag-resource \
+        --resource-arn arn:aws:cognito-idp:us-west-2:123456789012:userpool/us-west-2_EXAMPLE \
+        --tags administrator=Jie,tenant=ExampleCorp
+
+This command produces no output.
+
+For more information, see `Tagging Amazon Cognito resources<https://docs.aws.amazon.com/cognito/latest/developerguide/tagging.html>`__ in the *Amazon Cognito Developer Guide*.

```

Comment:
I think missing space between `resources` and `<https` is causing link rendering issues